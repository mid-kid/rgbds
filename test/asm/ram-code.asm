SECTION "test", ROM0[1]
	call Target
	LOAD "new", WRAM0[$C001]
Target:	dl $DEADBEEF
.end
	ENDL
After:
	jp Target
	ld hl, Word
	dw Byte, Target.end, After

SECTION "ram test", WRAM0 ; Should end up at $C005
Word:
	dw

SECTION "small ram test", WRAM0 ; Should end up at $C000
Byte:
	db

	PRINTT "{Target}\n{Target.end}\n{After}\n"
