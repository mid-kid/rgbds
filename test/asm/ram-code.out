$C001
$C005
$8
